src/esw/fw/imu/bno085_serial.ino

```diff
@@ -0,0 +1,154 @@
+#include <Adafruit_BNO08x.h>
+
+// For SPI mode, we need a CS pin
+#define BNO08X_CS 10
+#define BNO08X_INT 9
+
+// For SPI mode, we also need a RESET
+//#define BNO08X_RESET 5
+// but not for I2C or UART
+#define BNO08X_RESET -1
+
+Adafruit_BNO08x bno08x(BNO08X_RESET);
+sh2_SensorValue_t sensorValue;
+
+float rotationVector_i;
+float rotationVector_j;
+float rotationVector_k;
+float rotationVector_real;
```

Comment:
it would be nice to rename these to indicate its a quaternion and name the components xyzw to match how they are in ros and most other things